openwisp_monitoring/device/migrations/0001_initial.py

```diff
@@ -8,6 +8,11 @@ class Migration(migrations.Migration):
 
     initial = True
 
+    dependencies = [
+        ('contenttypes', '0002_remove_content_type_name'),
+        swapper.dependency('config', 'Device', '0004_add_device_model'),
+    ]
+
```

Comment:
Again, added for consistency. This file is not used in actual migration it has been squashed.